README.md

```diff
@@ -291,7 +291,6 @@ The [JAX image](https://github.com/NVIDIA/JAX-Toolbox/pkgs/container/jax) is emb
 | XLA Flags | Value | Explanation |
 | --------- | ----- | ----------- |
 | `--xla_gpu_enable_latency_hiding_scheduler` | `true`  | allows XLA to move communication collectives to increase overlap with compute kernels |
-| `--xla_gpu_enable_triton_gemm` | `false` | use cuBLAS instead of Trition GeMM kernels |
```

Comment:
Should we keep a list of old flags? This is still needed in the latest release of JAX.
We mostly need to tell that with JAX 0.4.32, it isn't needed, but that version isn't released yet.

Reply:
I can add the comment, thanks!